client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/standard-agency-hosting/index.tsx

```diff
@@ -126,6 +127,17 @@ export default function StandardAgencyHosting( { onAddToCart }: Props ) {
 							'We aimed to provide clients with a reliable hosting service we could endorse without hesitation, ultimately resulting in satisfied clients. We found that service with WordPress.com.'
 						),
 					},
+					{
+						profile: {
+							name: 'Brian Lalli',
+							avatar: ProfileAvatar1,
+							title: 'President - Moon Rooster LLC',
```

Comment:
Should we translate this title? I mean, the `President - ` part seems translatable.

Reply:
Thanks for the review, @cleacos!

Agree. I have added the translations now: https://github.com/Automattic/wp-calypso/pull/93236/commits/c5441f6f4fe3b92fd45cbcd3448a3c2763b77d1b